projects/dsp-ui/src/lib/search/advanced-search/select-property/select-property.component.spec.ts

```diff
@@ -0,0 +1,290 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { Properties, SelectPropertyComponent } from './select-property.component';
+import { Component, EventEmitter, Inject, Input, OnInit, Output, ViewChild } from '@angular/core';
```

Comment:
I would clean up the unused imports (there are several components like this one)

Reply:
done in https://github.com/dasch-swiss/knora-ui-ng-lib/pull/107/commits/7843e940dba649ead2ffd824cf9ab79c37848a5f